contracts/mainnet/connectors/etherfi-mint/main.sol

```diff
@@ -0,0 +1,36 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.2;
+
+import {Helpers} from "./helpers.sol";
+import {Events} from "./events.sol";
+import {TokenInterface} from "../../common/interfaces.sol";
+
+contract EtherfiMintConnector is Helpers, Events {
+    function mintEethFromStEth(
+        uint256 depositAmount_
+    )
+        external
+        payable
+        returns (string memory _eventName, bytes memory _eventParam)
+    {
```

Comment:
Add logic where depositAmount sent is uint max. so fetch DSA balance in that case.